miner/payload_building.go

```diff
@@ -229,6 +229,8 @@ func (miner *Miner) buildPayload(args *BuildPayloadArgs) (*Payload, error) {
 				r := miner.generateWork(fullParams)
 				if r.err == nil {
 					payload.update(r, time.Since(start))
+				} else {
+					log.Debug("Error while generating work", "id", payload.id, "error", r.err)
```

Comment:
Are errors here expected ? If not, wouldn't `log.Error` or at least `log.Info` be more correct?

Reply:
I think it is not expected, but it could be, I don't know how it behaves if you try to build two blocks on different heads

Reply:
IMO let's do at least info

Reply:
I think we can go with Info, the error is actually not expected

Reply:
done